src/test/shell/testenv.sh

```diff
@@ -595,6 +595,7 @@ workspaces=()
 # Set-up a new, clean workspace with only the tools installed.
 function create_new_workspace() {
   new_workspace_dir=${1:-$(mktemp -d ${TEST_TMPDIR}/workspace.XXXXXXXX)}
+  # try ignoring output here.
```

Comment:
Are you going to implement the actual ignoring logic?

Reply:
Grr, forgot to actually push it.

Reply:
Haha, no worries!